admin/services/sas-token.service.js

```diff
@@ -49,11 +70,56 @@ const sasTokenService = {
 
     const sasToken = serviceImplementation.generateSharedAccessSignature(queueName, sharedAccessPolicy)
 
-    return {
+    const tokenObject = {
       token: sasToken,
       url: serviceImplementation.getUrl(queueName),
       queueName: queueName
     }
+
+    // Store the sasToken in redis
+    if (redisKeyName) {
```

Comment:
Isn't that if block redundant?
`redisKeyName` will always be set as the `redisKeyService.getSasTokenKey` returns a string regardless.

Reply:
Good catch